scripts/src/sanitycheckpr/sanitycheckpr.py

```diff
@@ -12,6 +12,8 @@
 
 
 ALLOW_CI_CHANGES = "allow/ci-changes"
+SEMVER_MATCH_EXPRESSION = "(?P<major>0|[1-9]\d*)\.(?P<minor>0|[1-9]\d*)\.(?P<patch>0|[1-9]\d*)(?:-(?P<prerelease>(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+(?P<buildmetadata>[0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?"
```

Comment:
this is for chart version why pre-release and build-metadata is the something we would support and why? Provide examples. I would stop at patch

Reply:
The failing case we have was for the chart verifier test chart which has a version of  1.1.0-v3.valid. The file pattern match worked but the report match failed. It was report only so this meant we processed it as a valid request with no content so we needed to address that.  This is fixed by making the file pattern and the report pattern match the same for the version directory. 

As we have a version directory it seemed wise to include all valid semver formats - I can't say a community chart would never include a patch version. Also if we are to restrict this to a subset of valid semver we should document that, but if you want me to make it more restrictive I can.

Reply:
But we will not see such version in an official submission right?